tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>I suggest we add a warning regarding Mixin classes - something like the following:</p><p> </p><p>The below code uses a design pattern known as <em>Mixin classes</em>. This is a technique that (in addition to other benefits) allows for sharing of code between classes without using traditional inheritance. While this is a powerful technique, it can also easily lead to confusion, bugs, and code of low quality. Be sure that you read up proper use of Mixin before taking it into use in your own code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I am supporting that. (Maybe especially because I have not heard of these classes before.)

Reply:
I also support @keileg suggestion.